README.md

```diff
@@ -102,7 +102,10 @@ Steps 7 (Define) and 8 (Code) can occur concurrently. This may happen when you a
 
 ## The Process
 
-This process is composed of 8 steps which are introduced below:
+This process is composed of 8 steps which are introduced below.
+
+> A recommended talk about the process is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by Eduardo da Silva. He explains the process in the context of typical phases of designing sociotechnical architectures. Eduardo groups the activities in four phases "Align & Understand", "Strategic architecture", "Strategy & Org design" and "Tactical architecture".
```

Comment:
I like the extended version from Eduardo 🙂 
What about this:
```suggestion
> A good talk that gives an overview of the process in the context of typical phases of designing sociotechnical architectures is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by [Eduardo da Silva](@emgsilva). Eduardo groups the activities of the process and its 8 steps in four distinct phases, namely: 
1. Align & Understand
2. Strategic Architecture
3. Strategy & Org Design
4. Tactical Architecture.
```

Reply:
Makes it clearer indeed @yellowbrickc, thank you! Additional to this, and as mentioned above, I would include the link to presentation deck into the "in four distinct phases" text (the link can be directly to this part of the presentation, namely: https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31).

Reply:
```suggestion
> A good talk that gives an overview of the process in the context of typical phases of designing sociotechnical architectures is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by [Eduardo da Silva](@emgsilva). Eduardo groups the activities of the process and its 8 steps in [four distinct phases](https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31), namely: 
1. Align & Understand
2. Strategic Architecture
3. Strategy & Org Design
4. Tactical Architecture.
```

What about this?

Reply:
👍 exactly, I think this is it! Thank you @yellowbrickc